content/media-player/tracking/setup.md

```diff
@@ -0,0 +1,767 @@
+
+{{% tab name="HTML5 media tracking JS plugin" %}}
+Install the `@snowplow/browser-tracker` and `@snowplow/browser-plugin-media-tracking` via npm, yarn or any other package manager of your choice. Example using `npm`:
+
+```bash
+npm install @snowplow/browser-tracker @snowplow/browser-plugin-media-tracking
+```
+{{% /tab %}}
+
+{{% tab name="Web – script tag" %}}
+Add the `sp.js` file in your project and add it to a link-accessible server directory e.g. `public/`. The latest version can be found **[here](https://github.com/snowplow/snowplow-javascript-tracker/releases) .**
+
+Add the below snippet to all of the pages you would like to use Snowplow tracking. **Make sure to update the {{Link to the sp.js file}} variable.**
+
+Place the `<script>` tag into the `<head>` element of your pages:
+
+```jsx
+<script type="text/javascript">
+  (function (p, l, o, w, i, n, g) {
+    if (!p[i]) {
+      p.GlobalSnowplowNamespace = p.GlobalSnowplowNamespace || [];
+      p.GlobalSnowplowNamespace.push(i);
+      p[i] = function () {
+        (p[i].q = p[i].q || []).push(arguments);
+      };
+      p[i].q = p[i].q || [];
+      n = l.createElement(o);
+      g = l.getElementsByTagName(o)[0];
+      n.async = 1;
+      n.src = w;
+      g.parentNode.insertBefore(n, g);
+    }
+  })(window, document, "script", "{{Link to sp.js file}}", "snowplow");
+</script>
+```
+{{% /tab %}}
+{{% tab name="iOS" %}}
+You can install the tracker using SPM as follows:
+
+1. In Xcode, select File > Swift Packages > Add Package Dependency.
+2. Add the URL where to download the library: [https://github.com/snowplow/snowplow-ios-tracker](https://github.com/snowplow/snowplow-ios-tracker)
+
+To learn about other options for installing the tracker (e.g. using CocoaPods or Carthage), [see the mobile tracker documentation](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/installation-and-set-up/) .
+
+{{% /tab %}}
+
+{{% tab name="Android" %}}
+The tracker can be installed using Gradle. Add the following to your `build.gradle` file:
+
+```jsx
+dependencies {
+  ...
+  // Snowplow Android Tracker
+  implementation 'com.snowplowanalytics:snowplow-android-tracker:5.+'
+  // In case 'lifecycleAutotracking' is enabled
+  implementation 'androidx.lifecycle-extensions:2.2.+'
+  ...
+}
+```
+{{% /tab %}}
+
+{{< /tabs >}}
+
+
+#### **Step 3:** Create and configure the tracker
+
+Next, you will need to create a configured tracker instance.
+
+{{< tabs groupId="tracking sdk" >}}
+
+{{% tab name="Media JS plugin" %}}
+*Installed with package manager:*
+
+>To add the `SnowplowMediaPlugin` on the JavaScript tracker, you should include it as shown below:
+>
+>```jsx
+>import { newTracker } from '@snowplow/browser-tracker';
+>import { SnowplowMediaPlugin } from '@snowplow/browser-plugin-media';
+>
+>newTracker('sp1', '{{collector_url}}', { 
+>    appId: 'my-app-id', 
+>    plugins: [ SnowplowMediaPlugin() ],
+>});
+>```
+
+*Installed with script tag:*
```

Comment:
I think we can remove all these script tag sections and keep only the "Installed with package manager" instructions which don't have to be indented anymore.